Lib/test/test_webbrowser.py

```diff
@@ -461,11 +461,25 @@ def test_parse_args_error(self):
             "https://example.com --new-window --new-tab",
             "https://example.com -n --new-tab",
             "https://example.com --new-window -t",
-            # Ensure ambiguous shortening fails
-            "https://example.com --new",
         ]:
+            with support.captured_stderr() as stderr:
+                with self.assertRaises(SystemExit):
+                    webbrowser.parse_args(shlex.split(command))
+                self.assertEqual(
+                    stderr.getvalue(),
+                    "usage: __main__.py [-h] [-n | -t] url\n"
+                    "__main__.py: error: argument -t/--new-tab: not allowed with argument -n/--new-window\n"
```

Comment:
I would prefer to have something less verbose: `re.escape('error: argument -t/--new-tab: not allowed with argument -n/--new-window')` maybe?

Reply:
How's about
```python
self.assertIn(
     'error: argument -t/--new-tab: not allowed with argument -n/--new-window',
     stderr.getvalue(),
)
```
?